libsplinter/Cargo.toml

```diff
@@ -41,7 +41,7 @@ diesel = { version = "1.0", features = ["r2d2", "serde_json"], optional = true }
 diesel_migrations = { version = "1.4", optional = true }
 futures = { version = "0.1", optional = true }
 hyper = { version = "0.12", optional = true }
-jsonwebtoken = { version = "6.0", optional = true }
+jsonwebtoken = "6.0"
```

Comment:
Should this still be optional, and only included if rest-api is enabled.

Reply:
Updated